modules/@apostrophecms/asset/index.js

```diff
@@ -162,7 +173,7 @@ module.exports = {
             for (const name of names) {
               const moduleDir = `${modulesDir}/${name}`;
               for (const dir of directories[name]) {
-                const srcDir = `${dir}/ui/${source}`;
+                const srcDir = `${dir}/${source}`;
```

Comment:
The moduleOverrides function, which is not a method so it's OK to change its semantics, no longer assumes everything is in `ui/`.

Reply:
This permits us to use it for `public` which is separate from the `ui/*` subdirs.